oval-schemas/windows-definitions-schema.xsd

```diff
@@ -3322,6 +3332,23 @@
                                                 <xsd:documentation>The subgroup_sid entity holds a string that represents the SID of particular subgroup in the specified group. This entity can be included multiple times in a system characteristic item in order to record that a group contains a number of different subgroups. Note that the entity_check attribute associated with EntityStateStringType guides the evaluation of entities like subgroup_sid that refer to items that can occur an unbounded number of times.</xsd:documentation>
                                           </xsd:annotation>
                                     </xsd:element>
+									  <xsd:element name="group" type="oval-def:EntityStateStringType" minOccurs="0" maxOccurs="1">
+										   <xsd:annotation>
+												<xsd:documentation>A string the represents the name of a particular group. In Windows, group names are case-insensitive. As a result, it is recommended that the case-insensitive operations are used for this entity. In a domain environment, groups should be identified in the form: "domain\group name". For local groups use: "computer name\group name". For built-in accounts on the system, use the group name without a domain.</xsd:documentation>
+										   </xsd:annotation>
+									  </xsd:element>
+									  <xsd:element name="user" type="oval-def:EntityStateStringType" minOccurs="0" maxOccurs="unbounded">
+										   <xsd:annotation>
+												<xsd:documentation>A string that represents the name of a particular user. In Windows, user names are case-insensitive. As a result, it is recommended that the case-insensitive operations are used for this entity. In a domain environment, users should be identified in the form: "domain\user name". For local users use: "computer name\user name". For built-in accounts on the system, use the user name without a domain.</xsd:documentation>
+												<xsd:documentation>If the specified group has more than one user as a member, then multiple user elements should exist. If the specified group does not contain a single user, then a single user element should exist with a status of 'does not exist'. If there is an error determining the users that are members of the group, then a single user element should be included with a status of 'error'.</xsd:documentation>
+										   </xsd:annotation>
+									  </xsd:element>
+									  <xsd:element name="subgroup" type="oval-def:EntityStateStringType" minOccurs="0" maxOccurs="unbounded">
```

Comment:
ibid

Reply:
Good catch, sorry for yet another round of updates and review.  I'll get better with this eventually